src/Components/bookHub/BookCard.css

```diff
@@ -0,0 +1,42 @@
+.library-display {
+    width: 90vw;
+    display: flex;
+    flex-flow: row wrap;
+    justify-content: flex-start;
+    font-family: Arial, Helvetica, sans-serif;
+    background-color: #282c34;
+    border-color: black;
+    border-style: solid;
+}
+
+.card {
+    margin: 10px 1.25% 10px 1.25%;
```

Comment:
When the page is too small, the author fields end up getting warped weirdly! To fix this you can add a minimum width here instead of just the percentage so that the books wrap rather than squeezing continuously.